docs/tutorials/google/calibration_api.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>"are tools to characterize sqrt(iswap) gates" --&gt; "a family of gates including sqrt(iswap)" The family is represented by cirq.PhasedFSimGate blah blah. sqrt(iswap) corresponds to (-pi/4, 0, 0, 0, 0)</p><p> </p><p> </p><p>"insertion of Z phases" -&gt; "insertion of Z gates" (or "ZPowGates" or "RzGates")</p><p> </p><p> </p><p>"decreasing the effects of two sources of <strong><em>coherent</em></strong> errors:" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done. Reads a lot better and is more precise, thanks!</p>